src/index.ts

```diff
@@ -26,6 +26,8 @@ export interface Server {
     on(event: "exit", listener: ExitListener): void;
     /** Fires when the server closes (exits + closes IO streams). */
     on(event: "close", listener: CloseListener): void;
+    /** The PID of the subprocess. */
+    pid?: number;
```

Comment:
Note: this is potentially undefined because it is on serverProc (in reality, it's going to be defined).